flask_restplus_patched/parameters.py

```diff
@@ -232,7 +237,7 @@ def add(cls, obj, field, value, state):
         raise NotImplementedError()
 
     @classmethod
-    def remove(cls, obj, field, state):
+    def remove(cls, obj, field, value, state):
```

Comment:
Let's put a comment here that this is to support the functionality for removing users from project memberships, as an example.